litestar/_signature/model.py

```diff
@@ -61,6 +61,7 @@ class ErrorMessage(TypedDict):
     key: NotRequired[str]
     message: str
     source: NotRequired[Literal["body"] | ParamType]
+    exc: NotRequired[Dict[str, Any]]
```

Comment:
I think a different name like `errors` or something may be better since `exc` is usually used to mean an exception. Wdyt?

Reply:
Good point. This will contain data of a single error and will be accessed with `validation_exception["extra"][0]["exc"]`.
How about `raw`, `detail` or `error_detail`?